controllers/datadogagent/feature/test/testsuite.go

```diff
@@ -98,16 +99,17 @@ func runTest(t *testing.T, tt FeatureTest, buildFunc feature.BuildFunc) {
 	var gotConfigure feature.RequiredComponents
 	var dda metav1.Object
 	var isV2 bool
+	featureOptions := &feature.Options{}
+	if tt.FeatureOptions != nil {
+		featureOptions = tt.FeatureOptions
+	}
+	featureOptions.Logger = logger
```

Comment:
Added this to be able to unit test behavior with certain feature options. Tried to maintain the same logger behavior. Let me know if not advisable for any reason 🙇

Reply:
Thanks for calling out, looks fine to me!